### PR TITLE
Added entity champion

### PR DESCRIPTION
**Pull request by danielnottingham**

### CODE REVIEW

File: db/migrate/20210113131639_create_champions.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateChampions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :champions do |t|
+      t.string :name
+      t.string :title
+      t.string :tags
```
[the remark this answers is not in the document]
Comment (danielnottingham) [reply db/migrate/20210113131639_create_champions.rb:6]: ok

File: spec/factories/champions.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :champion do
+    name { 'Teemo' }
+    title { 'O boneco do capeta.' }
+    tags { 'Atirador / Assassino' }
+    image { 'Teemo.png' }
```
[the remark this answers is not in the document]
Comment (danielnottingham) [reply spec/factories/champions.rb:6]: yes

[the remark this answers is not in the document]
Comment (danielnottingham) [reply spec/factories/champions.rb:6]: https://ddragon.leagueoflegends.com/cdn/img/champion/splash/Teemo_0.jpg

[the remark this answers is not in the document]
Comment (edimossilva) [reply spec/factories/champions.rb:5]: so cute

**Review by edimossilva — APPROVED** [review]

**Pull request merged**